portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,35 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    
+  
+ 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Leah!");
+    ArrayList<String> jsonData = new ArrayList<String>();  
+    jsonData.add("nice!");
+    jsonData.add("cool!");
+    jsonData.add("awesome!");
+    convertToJsonUsingGson(jsonData);
```

Comment:
It looks like you're ignoring the return value from this method!

Reply:
The fact that it was still outputting correctly caused me to miss this, but it should be fixed now